pygments/lexers/jsonnet.py

```diff
@@ -0,0 +1,160 @@
+from pygments.lexer import RegexLexer
+from pygments.token import (
+    Comment,
+    Keyword,
+    Name,
+    Number,
+    Operator,
+    Punctuation,
+    String,
+    Whitespace,
+)
+
+__all__ = ['JsonnetLexer']
+
+jsonnet_token_chars = r'[_A-Za-z0-9]'
+jsonnet_token = r'[_A-Za-z]' + jsonnet_token_chars + '*'
+jsonnet_function_token = jsonnet_token + r'(?=\()'
+
+
+comments = [
+    (r'//.*\n', Comment.Single),
+    (r'#.*\n', Comment.Single),
+    (r'/\*\*([^/]|/(?!\*))*\*/', String.Doc),
+    (r'/\*([^/]|/(?!\*))*\*/', Comment),
+]
+
+
+whitespace = (r'[\n ]+', Whitespace)
+
+
+keywords = '|'.join([
```

Comment:
This is a good candidate for the [words helper function](https://pygments.org/docs/lexerdevelopment/#handling-lists-of-keywords), which creates an optimized regex for a collection of words, like the keywords you have here.